commands/24plus.js

```diff
@@ -0,0 +1,23 @@
+const { SlashCommandBuilder } = require("@discordjs/builders");
+
+const MAX = 11;
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName("24plus")
+        .setDescription("Generates 4 random numbers from 0 to 12 and a random target from 0 to 100."),
```

Comment:
im being gaslit, this is non-inclusive on the 12 (`MAX = 11`) but the original uses inclusive 9 so this makes me think this should be inclusive as well

Reply:
Oh yea, it will get +1'd to remove the zero. So actually, this comment should be Creates 4 random numbers from 1 - 12.